src/Scene/NodeProcess.js

```diff
@@ -85,6 +85,7 @@ NodeProcess.prototype.subdivideNode = function subdivideNode(node, camera, param
                 /* specific params */
                 bbox: bboxes[i],
                 type: quadtree.type,
+                light: params.layersConfig.lightingLayers[0],
```

Comment:
I would instead do this 10 lines below:
```js
quadtree.scheduler.execute(command).then((child) => 
    child.material.setLightingOn(params.light.enable);
    child.material.uniforms.lightPosition.value = params.light.position;
```
this way TileProvider doesn't need to be aware of Lighting parameters